components/brave_wallet/browser/polkadot/polkadot_substrate_rpc_responses.idl

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
2025

Reply:
Good catch. Always the copy-paste errors. This is done.